Compute/tests/System/V1/SmokeTest.php

```diff
@@ -102,6 +106,115 @@ public function testInsertInstance(): void
         $this->assertEquals(self::$machineType, $instance->getMachineType());
     }
 
+    /**
+     * @depends testInsertInstance
+     */
+    public function testUpdateDescInstanceToEmpty()
+    {
+        $instance = self::$instancesClient->get(
+            self::$name,
+            self::$projectId,
+            self::ZONE
+        );
+        $this->assertEquals("test", $instance->getDescription());
+        $this->assertEquals("0", $instance->getScheduling()->getMinNodeCpus());
+        $instance->setDescription("");
```

Comment:
nit: use double quotes only when necessary:
```suggestion
        $this->assertEquals('test', $instance->getDescription());
        $this->assertEquals('0', $instance->getScheduling()->getMinNodeCpus());
        $instance->setDescription('');
```